packages/concurrency-utils/package.json

```diff
@@ -1,5 +1,5 @@
 {
   "name": "@cloudflight/concurrency-utils",
   "version": "0.0.1",
-  "type": "module"
+  "type": "commonjs"
```

Comment:
might be smart to publish this as UMD and ESM:
https://dev.to/ipreda/how-to-easily-create-js-libraries-compatible-with-esamdumdcjs-module-systems-1kfd

Reply:
This has nothing to do with that though.

Reply:
This only tells us if `require` or `import` exists.